src/PointwiseFunctions/AnalyticData/ForceFree/FfeBreakdown.cpp

```diff
@@ -0,0 +1,81 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "PointwiseFunctions/AnalyticData/ForceFree/FfeBreakdown.hpp"
+
+#include <cstddef>
+#include <memory>
+#include <pup.h>
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "PointwiseFunctions/InitialDataUtilities/InitialData.hpp"
+#include "Utilities/MakeWithValue.hpp"
+#include "Utilities/TaggedTuple.hpp"
+
+namespace ForceFree::AnalyticData {
+
+FfeBreakdown::FfeBreakdown(CkMigrateMessage* msg) : InitialData(msg) {}
+
+std::unique_ptr<evolution::initial_data::InitialData> FfeBreakdown::get_clone()
+    const {
+  return std::make_unique<FfeBreakdown>(*this);
+}
+
+void FfeBreakdown::pup(PUP::er& p) {
+  InitialData::pup(p);
+  p | background_spacetime_;
+}
+
+PUP::able::PUP_ID FfeBreakdown::my_PUP_ID = 0;
+
+tuples::TaggedTuple<Tags::TildeE> FfeBreakdown::variables(
+    const tnsr::I<DataVector, 3>& x, tmpl::list<Tags::TildeE> /*meta*/) {
+  auto result = make_with_value<tnsr::I<DataVector, 3>>(x, 0.0);
+  result.get(1) = 0.5;
+  result.get(2) = -0.5;
+  return result;
+}
+
+tuples::TaggedTuple<Tags::TildeB> FfeBreakdown::variables(
+    const tnsr::I<DataVector, 3>& x, tmpl::list<Tags::TildeB> /*meta*/) {
+  auto result = make_with_value<tnsr::I<DataVector, 3>>(x, 1.0);
+  for (size_t i = 0; i < result.get(0).size(); ++i) {
+    const double& xi = x.get(0)[i];
```

Comment:
name `x` -> `coords` and then `xi` -> `x`. `xi` is the logical coordinate, so I worry this is confusing.